src/plugins/gui/components/GuiFormField.tsx

```diff
@@ -12,8 +12,8 @@ interface Props {
   fieldType?: FormFieldType
   key?: string
   autofocus?: boolean
-  returnKeyType?: OutlinedTextInputReturnKeyType
-  fieldRef?: React.Ref<OutlinedTextInputRef> | undefined
+  returnKeyType?: FilledTextInputReturnKeyType
+  fieldRef?: React.Ref<FilledTextInputRef> | undefined
```

Comment:
Because you have changed the ref type, you also need to update `src/plugins/gui/scenes/AddressFormScene.tsx`which still uses `OutlinedTextInputRef`.